core/merkleTree.go

```diff
@@ -17,7 +17,7 @@ func init() {
 		panic(err)
 	}
 
-	DefaultHashes, err = GenDefaultHashes(100)
+	DefaultHashes, err = GenDefaultHashes(100, ZeroLeaf)
```

Comment:
No change is required here. Just want to point out that we call the zero/empty leaf differently between the contract and the client.
https://github.com/thehubbleproject/hubble-contracts/pull/376#issue-534377693